AdvancedApplications/ODBCApplication/README.md

```diff
@@ -1,20 +1,29 @@
-# Instructions on How to Run ODBC Applications And DB2CLI For IBM Db2 Event Store
+# ODBC Remote Application / DB2CLI Execution Steps:
 
-## ODBC Remote Application / DB2CLI Execution Steps:
+Prior to running the sample application you must first set up the environment to run a ODBC/DB2CLI application. You must have completed the following steps found [here](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/tree/master/AdvancedApplications#odbcdb2cli-setup). If you are using the Docker container, the instructions are executed inside the container. 
```

Comment:
> the instructions are executed inside the container

I would change for "this environment setup must be executed inside the container"

Reply:
I moved the docker container forward which is what I think you were saying.